lumenlearning/IMathAS
# Showitemcourse

# What

1) Go through assessments(models) and make  sure that all the query's and secure.
2) Added filtering to the assesments 
2) Look through the components/showitemcourse 
# Test/Look-through

1) Check the commits and make sure changes work and look secure. 
    `Function  deleteAssessmentById`
    `function setStartDate`
    `function getByCourseIdJoinWithSessionData`
*Maybe I did a little too much in case
2) Look through all other querys to make sure I didn't miss anything.
3) Make sure that everything that needs to be filtered is being filtered
